releng/org.xtuml.bp.mctools/build.properties

```diff
@@ -0,0 +1,11 @@
+bin.includes = feature.xml
+root.win32.win32.x86=common,\
+                     win.all
+root.win32.win32.x86_64=common,\
+                        win.all
+root.linux.gtk.x86=common,\
```

Comment:
leaving 32 bit windows here in case we want to turn it back on at the top level.